devtools/gradle/gradle/libs.versions.toml

```diff
@@ -1,6 +1,7 @@
 [versions]
 plugin-publish = "1.2.0"
 
+# updating Kotlin here makes QuarkusPluginTest > shouldNotFailOnProjectDependenciesWithoutMain(Path) fail
 kotlin = "1.8.10"
```

Comment:
@snazy would you have an idea as to why we cannot upgrade Kotlin here?

We end up with:

>  Cannot access 'org.jetbrains.kotlin.gradle.plugin.KotlinTargetsContainerWithPresets' which is a supertype of 'org.jetbrains.kotlin.gradle.dsl.KotlinTargetContainerWithPresetFunctions'. Check your module classpath for missing or conflicting dependencies

I thought that maybe we had conflicting Kotlin versions but I couldn't find anything tied to 1.8.10. Or maybe Gradle comes with Kotlin 1.8.10?

Reply:
Yea, I think that Gradle comes with that version. It should however only affect Quarkus' Gradle plugin itself, not the application code.